lib/sycamore/sycamore/connectors/pinecone/pinecone_writer.py

```diff
@@ -119,10 +119,10 @@ def from_doc(cls, document: Document, target_params: "BaseDBWriter.TargetParams"
         else:
             id = f"{document.parent_id}#{document.doc_id}"
         values = document.embedding
-        metadata = {
+        metadata: dict[str, Any] = {
             "type": document.type,
             "text_representation": document.text_representation,
-            "bbox": document.bbox.to_dict() if document.bbox else None,
+            "bbox": document.bbox.to_list() if document.bbox else None,
```

Comment:
why list? this will make it show up in pc as
```
bbox.0: 325,
bbox.1: 14364,
bbox.2: 264,
bbox.3: 436245,
``` 
rather than 
```
bbox.x1: 136y4,
bbox.y1: 364,
bbox.x2: 2634,
bbox.y2: 2643
```

Reply:
Made the fix in the reader.